src/core/control/xojfile/LoadHandler.cpp

```diff
@@ -1035,7 +1036,7 @@ void LoadHandler::parserText(GMarkupParseContext* context, const gchar* text, gs
         if (!handler->pressureBuffer.empty()) {
             if (handler->pressureBuffer.size() + 1 >= handler->stroke->getPointCount()) {
                 auto firstNonPositive = std::find_if(handler->pressureBuffer.begin(), handler->pressureBuffer.end(),
-                                                     [](double v) { return v <= 0; });
+                                                     [](double v) { return v <= 0 || v != v; });
```

Comment:
Same here

Reply:
Thanks for the review. I amended these changes to the corresponding commit.